components/app/project-detail/ProjectSingleLink.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <tr>
+    <td class="flex my-2">
+      <img class="mr-2" :src="icon" :alt="title" />
```

Comment:
suggestion (non-blocking): try to be more precise with the alt description here. It will help with SEO and more importantly, it will help with accessibility.